common/extlib/client-extension-manager.mjs

```diff
@@ -1,6 +1,14 @@
 // Copyright 2021 HITCON Online Contributors
 // SPDX-License-Identifier: BSD-2-Clause
 
+import _axios from 'axios';
+import ClientExtensionHelper from './client-extension-helper.mjs';
+
+const axios = _axios.create({
```

Comment:
Note that we've already added jQuery and it can do async request. It's up to you which one you want to use.